contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -539,21 +539,6 @@ public void testCreateProcess() {
         assertTrue(processInformation.dwProcessId.longValue() > 0);
     }
 
-    public void testGetEnvironmentVariable() {
```

Comment:
I’m ok with breaking it up by functional group, e.g. Kernel32XXXTest.java (Kernel32EnvironmentVarsTest.java in this case).  Just keep the prefix so it’s easy to see what belongs together.



> On Apr 29, 2015, at 6:59 AM, Lyor Goldstein <notifications@github.com> wrote:
> 
> In contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java:
> 
> > @@ -539,21 +539,6 @@ public void testCreateProcess() {
> >          assertTrue(processInformation.dwProcessId.longValue() > 0);
> >      }
> >  
> > -    public void testGetEnvironmentVariable() {
> 
> Kernel32 is very big - to have a single class test all of the API seems too much - that is why in my past submissions (as in this one) I try to group tests related to a specific topic into a separate class (see NamedPipeTest) - as its the case for this submission - see EnvironmentVarsTest.. Regarding your remark, please note that I did not remove it (I would never remove a test unless some other test covers it) - I moved it to the EnvironmentVarsTest which groups the test for "excercising" the environment variables API(s)
> 
> —
> Reply to this email directly or view it on GitHub.
>